src/main/java/com/contrastsecurity/sdk/ContrastSDK.java

```diff
@@ -1830,13 +1831,29 @@ public HttpURLConnection makeConnection(String url, String method) throws IOExce
       connection.setRequestProperty(RequestConstants.TELEMETRY_INTEGRATION_VERSION, version);
     }
 
+    connection.setRequestProperty(HttpHeaders.USER_AGENT, buildUserAgentString().toString());
+
     connection.setUseCaches(false);
     if (connectionTimeout > DEFAULT_CONNECTION_TIMEOUT)
       connection.setConnectTimeout(connectionTimeout);
     if (readTimeout > DEFAULT_READ_TIMEOUT) connection.setReadTimeout(readTimeout);
     return connection;
   }
 
+  private StringBuilder buildUserAgentString() {
```

Comment:
This builds a string that only contains the integration name and version. We need to also include:

* comments
* the Contrast Java SDK name and version
* the JRE platform name and version

For example:

```
User-Agent: contrast-maven-plugin/2.12 (Apache Maven 3.8.1) contrast-sdk-java/2.22 Java/11.0.6
```